### PR TITLE
[node] Support node retry to establish connection with postgres

### LINKED ISSUE #57: [node] Node initialization should await Postgres get ready
Current node image often failed during initialization, due to it start running before the Postgres ready to accept connection.

### PR DESCRIPTION
**Pull request by jiqiang90**
Close #57 , now node can make 5 attempts to connect to Postgres with time interval of 3 seconds.

### CODE REVIEW

File: packages/node/src/db/db.module.ts [hunk 1]
```diff
@@ -13,6 +12,35 @@ export interface DbOption {
...
+): Promise<void> {
+  if (numRetries > 0) {
+    try {
+      console.log(
+        'Try to connect to database: ',
+        numRetries,
+        'attempts remaining',
+      );
+      await sequelize.authenticate();
+      console.log('Connection has been established successfully.');
```
Comment (ianhe8x) [inline packages/node/src/db/db.module.ts:31]: Before this pr we don't info user when db connection is established, so I suggest we keep same behaviour

File: packages/node/src/db/db.module.ts [hunk 2]
```diff
@@ -13,6 +12,35 @@ export interface DbOption {
...
+    } catch (error) {
+      console.error('Unable to connect to the database:', error.message);
```
Comment (ianhe8x) [inline packages/node/src/db/db.module.ts:33]: combine these two logs together, e.g: 
```suggestion
      console.error('Unable to connect to the database, will retry: ', error.message);
```

File: packages/node/src/db/db.module.ts [hunk 3]
```diff
@@ -13,6 +12,35 @@ export interface DbOption {
...
 
+async function timeout(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function establishConnection(
+  sequelize: Sequelize,
+  numRetries,
+): Promise<void> {
+  if (numRetries > 0) {
```
Comment (ianhe8x) [inline packages/node/src/db/db.module.ts:23]: check left retries in the `catch` block

**Review by ianhe8x — CHANGES_REQUESTED** [review]

File: packages/node/src/db/db.module.ts (hunk 1 above)
Comment (jiqiang90) [reply packages/node/src/db/db.module.ts]: Fixed

File: packages/node/src/db/db.module.ts (hunk 3 above)
Comment (jiqiang90) [reply packages/node/src/db/db.module.ts]: Fixed

File: packages/node/src/db/db.module.ts (hunk 2 above)
Comment (jiqiang90) [reply packages/node/src/db/db.module.ts]: Fixed

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**